spec/services/authorship_migration/work_version_creation_migration_spec.rb

```diff
@@ -0,0 +1,212 @@
+      end
+
+      Timecop.freeze(Time.zone.local(2021, 1, 3)) do
+        @wvc1.update(position: 20)
+
+        @wvc2 = create(
+          :work_version_creation,
+          work_version: work_version,
+          actor: @act2,
+          alias: 'Original Alias for Actor 2',
+          position: 10
+        )
+      end
+
+      Timecop.freeze(Time.zone.local(2021, 1, 4)) do
+        @wvc_deleted = create(
+          :work_version_creation,
+          work_version: work_version,
+          actor: @act_deleted,
+          alias: 'Original Alias for Actor Deleted',
+          position: 10
+        )
+      end
+
+      Timecop.freeze(Time.zone.local(2021, 1, 5)) do
+        @wvc_deleted.destroy
+      end
+    end
+  end
+
+  describe '.call' do
+    def perform_call
+      described_class.call(work_version: work_version)
+    end
+
+    # Sanity checks
+    it 'sets up a sane test environment' do
+      expect(PaperTrail::Version.where(item_type: 'WorkVersionCreation').count)
+        .to eq 6
+
+      expect(@wvc1.versions.map { |v| v.created_at.to_date })
+        .to match_array([
+                          Time.zone.local(2021, 1, 1).to_date,
+                          Time.zone.local(2021, 1, 2).to_date,
+                          Time.zone.local(2021, 1, 3).to_date
+                        ])
+    end
+
+    context 'when on the happy path' do
+      it 'creates a corresponding number of Authorship records' do
+        expect { perform_call }
+          .to change(Authorship, :count)
+          .by(2) # Note, TWO, because one of the WVC's above is deleted
+      end
+
+      it 'recreates the editing history of the migrated WorkVersionCreations' do
+        perform_call
+        expect(PaperTrail::Version.where(item_type: 'WorkVersionCreation').count)
+          .to eq(PaperTrail::Version.where(item_type: 'Authorship').count)
+
+        authorship1 = Authorship.find_by(actor_id: @act1)
+        expect(authorship1.resource).to eq work_version
+        expect(authorship1.alias).to eq 'New Alias for Actor 1'
+        expect(authorship1.given_name).to eq @act1.given_name
+        expect(authorship1.surname).to eq @act1.surname
+        expect(authorship1.email).to eq @act1.email
+        expect(authorship1.position).to eq 20
+        expect(authorship1.actor_id).to eq @act1.id
+        expect(authorship1.instance_token).to be_present
+        expect(authorship1.created_at.to_date).to eq Time.zone.local(2021, 1, 1).to_date
+        expect(authorship1.updated_at.to_date).to eq Time.zone.local(2021, 1, 3).to_date
+
+        expect(authorship1.versions.length).to eq 3
+        v1, v2, v3 = authorship1.versions
+        expect(v1.whodunnit).to eq user.to_gid.to_s
+        expect(v1.event).to eq 'create'
+        expect(v1.changeset.slice(:alias, :actor_id, :position, :created_at, :updated_at))
+          .to eq(@wvc1.versions[0].changeset.slice(:alias, :actor_id, :position, :created_at, :updated_at))
```

Comment:
This one test is failing because the first changeset has `display_name` while the second has `alias`.  We we'll probably need to break this down into

```ruby
expect(v1.changeset[:display_name]).to eq(@wvc1.versions[0].changeset[:alias])
expect(v1.changeset[:actor_id]).to eq(@wvc1.versions[0].changeset[:actor_id])
expect(v1.changeset[:position]).to eq(@wvc1.versions[0].changeset[:position])
expect(v1.changeset[:created_at]).to eq(@wvc1.versions[0].changeset[:created_at])
expect(v1.changeset[:updated_at]).to eq(@wvc1.versions[0].changeset[:updated_at])
```